.editorconfig

```diff
@@ -9,3 +9,6 @@ indent_style = space
 endofline = lf
 trim_trailing_whitespace = true
 insert_final_newline = true
+
+[*.json]
+indent_size = 2
```

Comment:
Thanks for explaining. Please delete this setting. Other than this, your PR looks good to me.